services/core-web/src/components/mine/Reports/MineReportTable.js

```diff
@@ -62,7 +62,9 @@ const defaultProps = {
 export const MineReportTable = (props) => {
   const hideColumn = (condition) => (condition ? "column-hide" : "");
 
-  const userIsAdmin = useSelector((state) => userHasRole(state, USER_ROLES.role_admin));
+  const userIsAdministrative = useSelector((state) =>
+    userHasRole(state, USER_ROLES.role_mds_administrative_users)
```

Comment:
Just checking if the intention behind the ticket is to _switch_ the role required to access delete or if it should be to _add another_ role that can delete, did not seem obvious to me.

Reply:
@taraepp when a user has admin role, doesn't he has all the roles?

Reply:
@taraepp when a user has an admin role, doesn't he has all the roles?

Reply:
Had a discussion around this here: https://discord.com/channels/976597427421323265/1252363484776235009